### ISSUE TITLE
3rd day with update on issue

### ISSUE THREAD

**Issue by pallamf** [krzychu124/Cities-Skylines-Traffic-Manager-President-Edition#695]
### Describe the problem
Pallamf
Hi.  I went in today, and it locked up again when unpausing the game.  I think I know how to get it to work, and am sharing here for completeness.

The other day, I disabled 81 tiles.  Went in, the game worked, came off pause.  25 tiles were unlocked.
I added purchase it.  Went in, the game worked, came off pause.  25 tiles were unlocked.

oday, I tried to open the game, and it locked when trying to unpause. (File a) )

Then, I tried safe mode with all 3 buttons enabled.  The game came off pause, but only went for 1 day, then locked up (perhaps when the first car spawned? ) (File b) )

I then disabled 25 tiles.  Went in, the game worked, came off pause and continues to run.  25 tiles are unlocked.  (File c) )
I attached the 3 player files related to the above 3 attempts.

### Steps to reproduce

1. 
2. 
3. 

### Log files

a) 
[Player remove 81 - add purchase it.log](https://github.com/krzychu124/Cities-Skylines-Traffic-Manager-President-Edition/files/4189343/Player.remove.81.-.add.purchase.it.log)
b)
[Player Safe mode with purchase it.log](https://github.com/krzychu124/Cities-Skylines-Traffic-Manager-President-Edition/files/4189344/Player.Safe.mode.with.purchase.it.log)
c) 
[Player disabled Purchase It.log](https://github.com/krzychu124/Cities-Skylines-Traffic-Manager-President-Edition/files/4189345/Player.disabled.Purchase.It.log)

**Comment by aubergine10** [comment]
There's no errors in the logs, so I think we're going to have to try process of elimination to see if we can narrow down the cause.

I use almost all the same mods you use, so if it's mod related issue then it will be one of the mods you use that I don't use. Here's a list of those mods:

* Parallel Road Tool
* Resize It
* Roundabout Builder
* Smart Intersections
* Pause On Load
* Speed Slider
* Net Picker
* Network Extensions 2 (don't remove this)
* Mods List
* Undo Mod

Could you try disabling those mods (except NExt2), then exit to desktop, then relaunch + load your save and see what happens?

**Comment by aubergine10** [comment]
@pallamf Did you get chance to try without those mods?

**Comment by pallamf** [comment]
@aubergine10  It took me a while to figure it out.  I was not able to reproduce the locking right away.  The short answer is it's "Pause on Load", but only if I play right away.

If I start up and let the game sit for about 30-45 seconds before taking it off pause, it'll work.  If I try to play sooner than that, it doesn't work, or if I try to resume a game, it locks.

When I removed Pause on Load, CS plays as expected.
To confirm, I readded Pause on Load, and the locking behavior started again.

**Comment by pallamf** [comment]
@aubergine10 It took me a while to consistently reproduce the problem.  In short, it's "Pause on Load", but only if try to play within 30-45 seconds of loading.  

If I load a game with POL, and play right away, or resume, it locks.  But if I let a game sit for 30-45 seconds before taking it off pause, it'll work.

Once I figured that out, I removed POL, and the game functioned normally.
I added POL back in, and it locked up as previously described.
